mobify/platform-scaffold
# [IM Extensibility] Categories & products custom selectors

This PR adds selectors for custom content on category and product items. Our existing results and reducers should be sufficient to allow partners to add the content, we just need to let them select it off the state.

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-75
 **Linked PRs**: n/a

## Changes
- Added selector for category custom content
- Added selector for product custom content
- Added temporary test content

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Browse to a PLP
- You should see the text "category custom content" next to the category title
- Browse to a PDP
- You should see the text "product custom content" under the product title

## To Do Before Merging
- [x] Remove test content
